src/k2/cmd/tpcc/tpcc_client.cpp

```diff
@@ -350,6 +369,7 @@ class Client {
     ConfigVar<int> _num_concurrent_txns{"num_concurrent_txns"};
     ConfigVar<uint16_t> _delivery_txn_batch_size{"delivery_txn_batch_size"};
     ConfigVar<uint16_t> _districts_per_warehouse{"districts_per_warehouse"};
+    ConfigVar<k2::String> _txn_weights{"txn_weights"};
```

Comment:
related to the parser above - just grab it as a vector<int>

Reply:
will do

Reply:
done